CMakeLists.txt

```diff
@@ -52,6 +52,11 @@ set(CMAKE_CXX_EXTENSIONS False)
 
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libstdc++ -std=c++20")
```

Comment:
Is `-std=c++20` needed. We already set standard to 20.

Reply:
Yes, nevermind.